Process.c

```diff
@@ -864,8 +864,12 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
    case STATE:
       xSnprintf(buffer, n, "%c ", this->state);
       switch (this->state) {
```

Comment:
In the long term it'd be better to have a canonical set of state values (an enum), switch all the PROCESS_N_STATE macros to descriptive names (like RUN), have platform-specific code refresh the 'state' enum.  This will keep the htop core code cleaner and more platform-neutral once again.  But at this stage of a release, I think this smaller fix is best.

Reply:
Can you open an issue for this so we don't forget about this? Looks like a good first issue to me.